hyperhtml.js

```diff
@@ -30,7 +30,7 @@ var hyperHTML = (function (globalDocument) {'use strict';
       notAdopting = false;
       hyperHTML.apply(node, arguments);
       notAdopting = true;
-      return this;
+      return node;
```

Comment:
Nice, I was considering raising an issue today, as I noticed this wasn't returning what I expected for adopted nodes. 👍